styles/variables.scss

```diff
@@ -1,20 +1,31 @@
 //Common
 $background-color: #c6dcf9;
 $background-color-light: #e7ecfa;
+$background-color-dark: #4279c0;
 $text-color: #404049;
 $text-gray-color: #666668;
 $text-blue-color: #619de4;
 $shadow-color: #ccc5c5;
 
 //Font size
 $fontSize-280px-smaller: 14px;
+$fontSize-280px-h1: 22px;
+$fontSize-280px-icon: 26px;
+$fontSize-768px-h1: 32px;
+$fontSize-768px-smaller: 20px;
+$fontSize-768px-icon: 46px;
+$fontSize-768px: 26px;
 $fontSize-1024px-capitals: 36px;
 $fontSize-1024px: 32px;
+$fontSize-1024px-h1: 40px;
 $fontSize-1024px-smaller: 26px;
 $fontSize-1024px-button: 26px;
+$fontSize-1024px-icon: 62px;
 $fontSize-1200px-capitals: 24px;
 $fontSize-1200px: 18px;
 $fontSize-1200px-smaller: 16px;
+$fontSize-1200px-h1: 22px;
+$fontSize-1200px-icon: 38px;
```

Comment:
taki system wprowadza trochę roboty, bo teraz dla każdego elementu będziesz musiała mieć nową zmienną, a jak widzisz trochę już ich jest, to raz, wiele z nich ma powtarzającą się wartość, np. 22px, 26px.

```
You can take a few different approaches to naming text styles:

Sized-based naming system (XS, S, M, L, XL)
Semantic naming system that corresponds to respective html tags in production (caption, paragraph, h1, h2)
Descriptive or functional naming system that explains the styles' intended use (alert, modal-header, button-label)
```
design system: https://www.figma.com/best-practices/typography-systems-in-figma/ , https://bootcamp.uxdesign.cc/how-to-name-text-styles-4-practical-approaches-204452a4bce8